flaskbb/utils/helpers.py

```diff
@@ -127,19 +135,19 @@ def do_topic_action(topics, user, action, reverse):
         if not Permission(CanDeleteTopic):
             flash(
                 _("You do not have the permissions to delete these topics."),
-                "danger"
+                "danger",
```

Comment:
[`flash` takes 2 arguments at most](https://github.com/pallets/flask/blob/1fa9185c7e82ecc43b99d797af5584133458008d/flask/helpers.py#L386), the comma is unnecessary and confusing.